src/alto-ui/Tag/__tests__/Tag.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Tag from './../Tag';
+
+describe('Tag', () => {
+  const getWrapper = props => shallow(<Tag {...props} />);
+
+  const defaultProps = {
+    className: 'className',
+    active: true,
+    rounded: true,
+    propMock: true,
+    onClick: jest.fn(),
+    disabled: true,
+  };
+
+  it('should render without error', () => {
+    const wrapper = getWrapper({});
+
+    expect(wrapper.find('div')).toHaveLength(1);
+    expect(wrapper.find('div').hasClass('Tag')).toBe(true);
+  });
+
+  it('should render button if passed onClick prop', () => {
+    const wrapper = getWrapper(defaultProps);
+
+    expect(wrapper.find('button')).toHaveLength(1);
+  });
+
+  it('should set correct class names', () => {
+    const wrapper = getWrapper(defaultProps);
```

Comment:
use just `const wrapper = getWrapper(defaultProps).find('button');` and remove line 34

Reply:
Did this and I named there the variable button:
`const button = getWrapper(defaultProps).find('button');`